src/tests/Microsoft.PowerFx.Core.Tests/Helpers/BombCulture.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using System.Text;
+
+namespace Microsoft.PowerFx
+{
+#pragma warning disable CA1065 // Do not raise exceptions in unexpected locations
+
+    // Used to verify that that we're not accessing this culture. 
+    // For example, if we want to verify we don't access Thread.CurrentCulture. 
+    // then just set it to BombCulture and any access will throw. 
+    [DebuggerDisplay("Bomb culture")]
+    public class BombCulture : CultureInfo
```

Comment:
I like this technique - adding a comment here to draw attention to it.  It made some surprising discoveries  (see changes in \Functions\Library*.cs)

Reply:
I don't like "bomb" term or any war reference
I think InvalidCulture is sufficient